tutor/src/components/reference-book/page-navigation.cjsx

```diff
@@ -5,33 +5,55 @@ React = require 'react'
 classnames = require 'classnames'
 Router = require 'react-router'
 
-PageNavigation = React.createClass
+PagingNavigation = require '../paging-navigation'
+{ReferenceBookStore} = require '../../flux/reference-book'
+{ChapterSectionMixin} = require 'shared'
+
+RefenceViewPageNavigation = React.createClass
```

Comment:
Ug. Auto-complete is so nice until you speeell something wrong once and then manage to do the same thing all over the place.  Thanks!